.github/workflows/update-oneclick-apps.yml

```diff
@@ -0,0 +1,24 @@
+name: Update one click apps
+on:
+  release:
+    types: [published]
+jobs:
+  update-digitalocean-oneclick-app:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - name: Update DigitalOcean one click app
+        env:
+          DIGITALOCEAN_API_KEY: ${{ secrets.DIGITALOCEAN_API_KEY }}
+          ZULIP_API_KEY: ${{ secrets.ZULIP_API_KEY }}
+          ZULIP_EMAIL: ${{ secrets.ZULIP_EMAIL }}
+          ZULIP_SITE: ${{ secrets.ZULIP_SITE }}
+          ZULIP_STREAM: ${{ secrets.ZULIP_STREAM }}
```

Comment:
We may want to make this secret be a little more explicit -- zulip stream for _what_?

Reply:
Made the secret names more explicit